dali/test/python/test_operator_python_function.py

```diff
@@ -319,3 +332,24 @@ def test_wrong_outputs_number():
         print(e)
         return
     raise Exception('Should not pass')
+
+
+SINK_PATH = '/tmp/sink_test' + str(SEED) + '/'
```

Comment:
How about `tempfile.mkdtemp`? Just in case previous test failed and the path was not removed.

Reply:
done